fedimint-client-module/src/module/init.rs

```diff
@@ -186,6 +186,8 @@ where
     }
 
     pub fn update_recovery_progress(&self, progress: RecoveryProgress) {
+        // we want a warning if the send channel was not connected to
```

Comment:
you can just use `receiver_count` for the warning

Reply:
How is that better? `send` seems to have exactly the semantic we want, no?

Reply:
https://github.com/fedimint/fedimint/pull/7067#issuecomment-2741488198

Reply:
storing the latest value in watch doesn't hurt us